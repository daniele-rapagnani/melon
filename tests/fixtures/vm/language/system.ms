if (system.getArchBits() == 64 || system.getArchBits() == 32) {
    io.print("Valid architecture");
}

if (#system.getMelonVersion() > 0) {
    io.print("Valid melon version");
}