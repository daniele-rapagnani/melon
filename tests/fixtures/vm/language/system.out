Valid architecture
Valid melon version
